crates/sui-indexer/tests/integration_tests.rs

```diff
@@ -117,7 +93,69 @@ mod pg_integration {
                 Some(SuiTransactionResponseOptions::full_content()),
                 Some(ExecuteTransactionRequestType::WaitForLocalExecution),
             )
-            .await?;
+            .await
+            .map_err(|e| anyhow::anyhow!(e))?;
+        Ok((tx_response, *sender, *recipient, gas_objects))
+    }
+
+    #[tokio::test]
+    async fn test_genesis_sync() {
```

Comment:
this is a bit misleading, I moved another block of codes to `execute_simple_transfer` actually